app/components/donation/donation-container.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const {
+  Component,
+  computed: { bool }
+} = Ember;
+
+export default Component.extend({
+  classNames: ['donation-container'],
+  donationAmount: 0,
+  projectTitle: null,
+
+  canDonate: bool('projectTitle')
```

Comment:
My assumption is that checking for the existence of a project would happen at the route or controller level and this component would only ingest the title.  The action of donation would bubble up to the parent, so we shouldn't have a need for the entire model here.